alkira/resource_alkira_credential_azure_vnet.go

```diff
@@ -9,47 +9,48 @@ import (
 
 func resourceAlkiraCredentialAzureVnet() *schema.Resource {
 	return &schema.Resource{
-		Description: "Manage AWS credential for authentication.\n\n" +
-			"You can provide your credentials via the `AK_AZURE_APPLICATION_ID` " +
-			"`AK_AZURE_SUBSCRIPTION_ID`, `AK_AZURE_SECRET_KEY` and " +
-			"`AK_AZURE_TENANT_ID` environment variables.",
+		Description: "Credential for interacting with Azure.\n\n" +
+			"You could also provide your credentials via the following " +
+			"environmental variables:\n\n * AK_AZURE_APPLICATION_ID\n " +
+			"* AK_AZURE_SUBSCRIPTION_ID\n * AK_AZURE_SECRET_KEY\n " +
+			"* AK_AZURE_TENANT_ID\n",
 		Create: resourceCredentialAzureVnet,
 		Read:   resourceCredentialAzureVnetRead,
 		Update: resourceCredentialAzureVnetUpdate,
 		Delete: resourceCredentialAzureVnetDelete,
 
 		Schema: map[string]*schema.Schema{
 			"name": &schema.Schema{
-				Description: "The name of the credential",
+				Description: "The name of the credential.",
```

Comment:
This is a little thing but there is inconsistency here. Some of the fields include 'The' as the first word in the descriptor. We either need it to be 'The' for all of them or in this case 'Name of the credential.' I don't particularly care which though I think removing 'The' is more succinct and less repetitive. So yes, lets remove 'The' as a general rule.

Reply:
Oh hahaha I just saw you did that below. Sorry. You just missed this 'The'.

Reply:
Good point! Hehe, I was puzzled too why `The`.